tests/algorithms/extended_temporal_memory/tm_unit_test_base.py

```diff
@@ -343,7 +344,7 @@ def testNoChangeToMatchingSegmentsInPredictedActiveColumn(self):
 
 
     tm.compute(previousActiveColumns)
-    self.assertEqual(expectedActiveCells, tm.getPredictiveCells())
+    np.testing.assert_equal(expectedActiveCells, tm.getPredictiveCells())
```

Comment:
This was part of the commit "ETM unit tests: don't fail on numpy arrays".  `self.assertEqual([1,2,3], np.array([1,2,3]))` will fail because only one of them is a list. `np.testing.assert_equal` will compare lists to numpy arrays without complaining.